facade/nodeFacade.go

```diff
@@ -190,82 +177,6 @@ func (nf *nodeFacade) RestApiInterface() string {
 	return nf.config.RestApiInterface
 }
 
-func (nf *nodeFacade) startRest() {
-	log.Trace("starting REST api server")
-
-	switch nf.RestApiInterface() {
-	case DefaultRestPortOff:
-		log.Debug("web server is off")
-	default:
-		log.Debug("creating web server limiters")
-		limiters, err := nf.CreateMiddlewareLimiters()
-		if err != nil {
-			log.Error("error creating web server limiters",
-				"error", err.Error(),
-			)
-			log.Error("web server is off")
-			return
-		}
-
-		log.Debug("starting web server",
-			"SimultaneousRequests", nf.wsAntifloodConfig.SimultaneousRequests,
-			"SameSourceRequests", nf.wsAntifloodConfig.SameSourceRequests,
-			"SameSourceResetIntervalInSec", nf.wsAntifloodConfig.SameSourceResetIntervalInSec,
-		)
-
-		srv, err := api.CreateServer(nf, nf.apiRoutesConfig, limiters...)
-
-		if err != nil {
-			log.Error("could not create webserver", "error", err.Error())
-		}
-
-		go func() {
-			err = srv.ListenAndServe()
-			if err != nil {
-				if err != http.ErrServerClosed {
-					log.Error("could not start webserver",
-						"error", err.Error(),
-					)
-				} else {
-					log.Debug("ListenAndServe - webserver closed")
-				}
-			}
-		}()
-
-		nf.server = srv
-	}
-}
-
-// CreateMiddlewareLimiters will create the middleware limiters used in web server
-func (nf *nodeFacade) CreateMiddlewareLimiters() ([]api.MiddlewareProcessor, error) {
```

Comment:
maybe remove this function from the disabled implementation also?

Reply:
already removed